orchestrator/dockerdir/usr/bin/add_mysql_nodes.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+set -o xtrace
+
+ORC_HOST=127.0.0.1:3000
+
+wait_for_leader() {
+    local retry=0
+    local leader=""
+
+    until [ ${leader} -ne "" ]; do
+        if [ ${retry} -gt 60 ]; then
+            echo "Waiting for leader failed after 60 attempts"
+            exit 1
+        fi
+
+        local leader=$(curl "${ORC_HOST}/api/raft-leader" 2>/dev/null)
+
+        retry=$(($retry + 1))
+        sleep 1
+    done
+}
+
+am_i_leader() {
+    local resp=$(curl -w httpcode=%{http_code} "${ORC_HOST}/api/leader-check" 2>/dev/null)
+    local http_code=$(echo "${resp}" | sed -e 's/.*\httpcode=//')
```

Comment:
why do we need to have  it? We can add additional 'sed' to the 'curl -w httpcode=%{http_code} "${ORC_HOST}/api/leader-check"'

Reply:
https://github.com/percona/percona-docker/pull/496/commits/fe30d4e90a25a76cb68af0a615edc86ec408f8b6